### PR TITLE
Add Bits Innovation and change ebPearls description

### PR DESCRIPTION
**Pull request by girisagar46**

### CODE REVIEW

**Comment by mesaugat** [comment]
Hello @girisagar46 - We already have `EB Pearls` in the list. If you want to change the description or the title please make your branch up-to-date with master and push the changes again.

**Comment by mesaugat** [comment]
@girisagar46 Can you fix the conflicts and push again?

File: README.md [hunk 1]
```diff
@@ -21,12 +21,13 @@ If you cannot provide a valid website for your company your PR will not be accep
 - [Avionte Solutions](http://www.avionte.com/) | Buddhanagar, Kathmandu | A robust staffing solution
 - [Bidhee Pvt. Ltd.](http://www.bidhee.com) | New Baneshwor, Kathmandu | A company solely established for development and deployment of innovative products and services in the field of IT globally
 - [big B soft](http://bigbsoft.com) | Nayabazar, Kathmandu | Native - iOS app development agency
+- [Bits Innovation](http://bitsinnovation.com/) | New Baneshwor | Provides service like Website Design & Development, Search Engine Optimization, Mobile Apps Development and Web hosting
```
Comment (mesaugat) [inline README.md]: Let's make it `New Baneshwor, Kathmandu`

File: README.md [hunk 2]
```diff
@@ -21,12 +21,13 @@ If you cannot provide a valid website for your company your PR will not be accep
...
 - [Braindigit](http://www.braindigit.com/) | Gyaneshwor, Kathmandu | Building applications for businesses and mobile on a native platform, with driven project teams you can call yours
 - [CloudFactory](https://www.cloudfactory.com/) | Bhainsepati, Lalitpur | On-Demand Workforce for Data Work
 
 ## D - F
 - [Deerwalk](http://www.deerwalk.com/) | Sifal, Kathmandu | Bringing you software solutions for POPULATION HEALTH MANAGEMENT
-- [EB Pearls](https://www.ebpearls.com.au/) | Kupondole, Lalitpur | Digital agency for web design, ecommerce and mobile app development
+- [ebPearls Pvt. Ltd](http://www.ebpearls.com/) | Kupondole, Lalitpur | EB Pearls plans, designs and manages its client’s digital product in Sydney office while the development and quality controlling is executed in Kathmandu office.
```
Comment (mesaugat) [inline README.md]: Can we make the description focus on what the company does and not where it is based? I find the previous description much better.

Also, do remove the full stop at the end.

Comment (mesaugat) [inline README.md]: We will also need a full stop after `Ltd`

Comment (girisagar46) [inline README.md]: ok. I'll do that in new pull requests.

Comment (mesaugat) [inline README.md]: You don't need to make a new pull request. Just push the new changes here.

Comment (mesaugat) [inline README.md]: @girisagar46 Did you miss this one?

Comment (girisagar46) [inline README.md]: Please close this PR. I don't know what went wrong and I messed up. There's merge conflict.

Comment (girisagar46) [inline README.md]: I think [this](https://github.com/mesaugat/tech-companies-in-nepal/pull/9/commits/43e3ce62c2c1873328a108ac9e915bb066fc4ae9) commit is clean one

**Comment by mesaugat** [comment]
@girisagar46 I'll close this for now. Do send in a new pull request. Make sure your master branch is up to date.

**Comment by mesaugat** [comment]
@girisagar46 You should add `Bits Innovation` to the list in a new PR. Thanks.

**Comment by mesaugat** [comment]
@girisagar46 I've added `Bits Innovation` to the list. Check #18

**Pull request closed without merging**